src/sentry/options/defaults.py

```diff
@@ -577,6 +577,11 @@
 # values or not
 register("sentry-metrics.performance.index-tag-values", default=True)
 
+
+# A slow rollout option for writing "new" cache keys
+# as the transition from UseCaseKey to UseCaseID occurs
+register("sentry-metrics.indexer.cache-key-rollout-rate", default=0.0)
```

Comment:
Will this option control the rollout for both generic metrics and release health pipelines? Asking since I am not sure whether this would impact both the pipelines at once.

Reply:
Yes, we're thinking it would. 

Alternatively, we could also make it such that the cache keys for release health stay the same, but then we will have some inconsistency where the cache keys contain the use_case_id/namespace on one pipeline (generic metrics) and the metric path key on another (release health).

Reply:
Yep, that makes sense.

Reply:
👍